src/view.c

```diff
@@ -284,6 +284,29 @@ view_set_activated(struct view *view, bool activated)
 			keyboard_update_layout(&view->server->seat, view->keyboard_layout);
 		}
 	}
+	view_set_adaptive_sync_fullscreen(view);
+}
+
+void
+view_set_adaptive_sync_fullscreen(struct view *view)
```

Comment:
We prefix view functions that are called from other translation units with `_view`. 
It's okay to drop the `_view` prefix when it's a local function... so up to you.